lighthouse-core/gather/gather-runner.js

```diff
@@ -492,7 +492,7 @@ class GatherRunner {
         msg: `Gathering: ${gatherer.name}`,
         id: `lh:gather:afterPass:${gatherer.name}`,
       };
-      log.time(status);
+      log.time(status, 'log');
```

Comment:
this jumped out at me as being weird, so flagged our level types for it. how would you feel about an alias for `.log` `.info` so we can use the real level here?